source/Missions/Services/Arena/CoopArenaController.cs

```diff
@@ -120,10 +164,9 @@ public void AddPlayerToArena()
 
             Agent.Main.SetTeam(Mission.Current.PlayerTeam, false);
 
-            IEnumerable<CharacterObject> listC = CharacterObject.All.Where(x => !x.IsHero);
-            Random r = new Random();
+            IEnumerable<CharacterObject> listC = CharacterObject.All.Where(x => !x.IsHero && !x.IsChildTemplate);

```

Comment:
Move this to the constructor and assign it to a class variable. Also make this an array for quicker access